components/payment_form/gather_intent/gather_intent.scss

```diff
@@ -0,0 +1,171 @@
+@import 'utils/mixins';
+
+.gatherIntent {
+    margin-bottom: 24px;
+
+    &__title {
+        padding: 0;
+        margin: 0 0 8px;
+        font-size: 16px !important;
+        font-weight: 600 !important;
+        line-height: 24px !important;
+    }
+
+    &__button {
+        height: auto !important;
+        padding: 0 !important;
+        border: none !important;
+        background: transparent !important;
+        color: var(--button-bg) !important;
+        font-size: 14px !important;
+        font-weight: 400 !important;
+        line-height: 20px !important;
+    }
+}
+
+.savedFeedback__text {
+    align-self: center;
+    color: rgba(63, 67, 80, 1);
+    font-size: 22px;
+    font-weight: 600;
+}
+
+.GatherIntentModal {
+    .modal-content {
+        max-width: 512px;
+        max-height: 465px;
+        border-radius: 12px;
+    }
+
+    .modal-dialog {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        min-width: 532px;
+        margin: auto;
+        transform: translate(-50%, -50%) !important;
+    }
+
+    .modal-footer {
+        padding-top: 24px;
+        padding-bottom: 24px;
+        border-radius: 0 0 12px 12px;
+    }
+
+    .modal-header {
+        border-radius: 12px 12px 0 0;
+    }
+
+    .modal-header,
+    .modal-footer {
+        display: flex;
+        padding-right: 32px;
+        padding-left: 32px;
+    }
+
+    .GatherIntentModal__header {
+        align-items: baseline;
+        border: none;
+
+        &.modal-header {
+            background: var(--center-channel-bg);
+        }
+
+        h3 {
+            margin: 0;
+            color: rgba(63, 67, 80, 1);
```

Comment:
I think instances of 63, 67, 80 can be replaced with ` --center-channel-text-rgb`.